photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditor.java

```diff
@@ -383,11 +375,18 @@ public PhotoEditor build() {
     }
 
 
+    // region Shape
     /**
-     * Enumerates the different shapes that can be drawn on a brush surface.
+     *
      */
-    enum BrushDrawingShape {
-        FREE_HAND, OVAL, RECTANGLE
-    }
+    void addShape(Shape shape);
```

Comment:
We can use `ShapeBuilder` pattern as we did for `TextStyleBuilder` which provide more flexibility to add more properties to shape. And will also avoid to have this huge interface with too many methods.